tests/mocha/index.html

```diff
@@ -32,6 +32,17 @@
       <block type="basic_block"></block>
     </xml>
 
+    <xml id="toolbox-categories" style="display: none">
+      <category name="First">
+        <block type="basic_block">
+          <field name="TEXT">First</field>
+        </block>
+      </category>
+      <category name="Second">
+        <field name="TEXT">Second</field>
```

Comment:
field isn't a valid tag inside a category. Did you mean to have another basic_block here or for it to be a label?

Reply:
Meant to have a basic_block.  Fixed.